OpenEdXMobile/src/test/java/org/edx/mobile/view/WebViewActivityTest.java

```diff
@@ -53,11 +58,20 @@ protected static void test_StartWebViewActivity_LoadsUrlAndShowsTitle(@NonNull S
         assertEquals(shadowWebView.getLastLoadedUrl(), url);
         final ActionBar actionBar = activity.getSupportActionBar();
         assertNotNull(actionBar);
-        assertTrue(actionBar.isShowing());
+        assertThat(actionBar).isShowing();
         if (!TextUtils.isEmpty(title)) {
-            assertTrue(title.equals(actionBar.getTitle()));
+            assertThat(actionBar).hasTitle(title);
+        }
+        /*
+        Robolectric is not providing the correct default title which is why this code has
+        been commented.
+
+        else {
+            final PackageManager pm = activity.getPackageManager();
```

Comment:
Code at the same level as comment without explicit comment indicator at start looks a bit weird, but I suppose it's the most convenient way to do it.